consensus/polybft/bridge_event_manager.go

```diff
@@ -322,8 +322,8 @@ func (b *bridgeEventManager) getAggSignatureForBridgeBatchMessage(blockNumber ui
 		return Signature{}, err
 	}
 
-	signatures := make(bls.Signatures, 0, len(votes))
-	publicKeys := make([][]byte, 0, len(votes))
+	var signatures bls.Signatures
```

Comment:
Why it is not ok to initialize the capacity to the worst case scenario?

```suggestion
	signatures := make(bls.Signatures, 0, len(votes))
```

Reply:
I think it is ok. I will add that.

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/372/commits/a3d92319cc9d88b784264c3fa6733e9c6423eb44